booking.html

```diff
@@ -1,66 +1,104 @@
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Booking page</title>
+    <link rel="stylesheet" href="css/booking.css" />
+    <link rel="icon" href="./favicon.ico" type="image/x-icon" />
+</head>
+
+<body>
+    <!--Header and nav starts here -->
+    <header>
+        <nav>
+            <ul>
+                <li><a href="#">Home</a></li>
+                <li><a href="#">Gallery</a></li>
+                <li><a href="#">Price</a></li>
+                <li><a href="#">Book sessions</a></li>
+                <li><a href="#">About me</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul>
+        </nav>
+    </header>
+    <!--header and nav ends here -->
+    <main>
+        <h1>Book sessions with us!</h1>
+        <form class="booking-form">
+            <div class="name-group">
+                <p class="name-group-title">Client´s Name</p>
+                <div class="name-inputs">
+                    <label for="firstName">First Name:</label>
+                    <input type="text" id="firstName" name="firstName" required placeholder="First name" />
                     <label for="lastName">Last Name:</label>
-                    <input
-                        type="text"
-                        id="lastName"
-                        name="lastName"
-                        required
-                        placeholder="Last name"
-                    />
+                    <input type="text" id="lastName" name="lastName" required placeholder="Last name" />
                 </div>
+            </div>
 
+            <div class="mail-tab">
+                <label for="email">Email:</label>
+                <input type="email" id="email" name="email" required placeholder="Add your email" />
+            </div>
+            <div>
+                <label for="date">dd/mm/yyy:</label>
+                <input type="date" id="date" name="date" required placeholder="Date" </div>
                 <div>
-                    <label for="email">Email:</label>
-                    <input
-                        type="email"
-                        id="email"
-                        name="email"
-                        required
-                        placeholder="Add your email"
-                    />
+                    <label for="time">set Time:</label>
+                    <textarea id="time" name="time" required placeholder="set Time"></textarea>
                 </div>
-
                 <div>
                     <label for="message">Message:</label>
-                    <textarea
-                        id="message"
-                        name="message"
-                        rows="3"
-                        required
-                        placeholder="Extra information"
-                    ></textarea>
+                    <textarea id="message" name="message" rows="3" required placeholder="Extra information"></textarea>
```

Comment:
The `id` should probably be unique. It is reused a couple of times in this file.

Reply:
cant see `message` being used anywhere, so maybe it can be removed.